Functions/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v2.10.0
+- [added] Add a contstructor to set a custom domain. (#6787)
```

Comment:
Release notes should be past tense, and use `class method` instead of `constructor`.

Reply:
But it's a constructor, isn't it?  I was following the wording from the release note about setting region.

Reply:
Alright then, keep your constructors. But do fix the spelling!